mmdet/utils/misc.py

```diff
@@ -21,3 +26,23 @@ def __call__(self, parser, namespace, values, option_string=None):
                     val = val[0]
             options[key] = val
         setattr(namespace, self.dest, options)
+
+
+def wrap_model_with_mmdet(model, cfg, distributed=False):
```

Comment:
I don't like the name. Saying that mmdet model is wrapped with mmdet does not provide any useful hints on what is actually done. `wrap_model_with_data_parallel` looks more descriptive, though better naming idea are appreciated.

Reply:
Changed to `prepare_mmdet_model_for_execution` since it could wrap the model with custom mmdet DP/DDP/CPU wrappers.